plugin.php

```diff
@@ -487,3 +487,16 @@ function rest_is_boolean( $maybe_bool ) {
 		return false;
 	}
 }
+
+/**
```

Comment:
@joehoyle @rmccue I think you were one step ahead of me, I just updated develop and see the new function. updating the PR, all I have to add now is the new tests, everything checks out with this fix in place.